### ISSUE TITLE
Multiple Content-Transfer-Encoding headers causes Exception

### ISSUE THREAD

**Issue by fko2221** [php-mime-mail-parser/php-mime-mail-parser#126]
A message that has more than one Content-Transfer-Encoding header causes the following Exception to be thrown:

strtolower() expects parameter 1 to be string, array given in Parser.php on line 535

I know there's only supposed to be one Content-Transfer-Encoding header but I would like to handle this gracefully instead of throwing an exception. 

A sample message is attached.
[message.txt](https://github.com/php-mime-mail-parser/php-mime-mail-parser/files/598151/message.txt)

I will submit a PR for this issue.

**Comment by dikirill** [comment]
@eXorus 
Hi, I have the same error, but coming from another place:
```
PHP Warning: strtolower() expects parameter 1 to be string, array given in php-mime-mail-parser/php-mime-mail-parser/src/Parser.php on line 580
```
So, if you add
```
if (is_array($encodingType)) {
   $encodingType = $encodingType[0];
}
```
to the line Parser.php:542 will help.
To reproduce, use the same example message.txt.

**Comment by eXorus** [comment]
Thanks fixed in #198 and released in 2.11.1
